### DISCUSSION TITLE
Using ASReview and Covidence Together

### DISCUSSION THREAD

**Discussion by dchamn** [asreview/asreview#1601] [Q&A]
Hi all,

My team and I will be using Covidence for our review, but want to use ASReview to help us cut down on screening during abstract/title screening. Is it possible to import an ASReview file with reviewer's decisions into Covidence and have Covidence read the decisions (include/exclude) about each abstract?

Thanks!

**Comment by FelixWdm** [comment]
Hi @dchamn , I can't fully check this at the moment, we no longer have access to Covidence unfortunately. You could choose to just export the relevant records, and load these into Covidence. 
![afbeelding](https://github.com/asreview/asreview/assets/49488769/82cf07d9-4024-4b9c-8c83-a3debb1ce6e1)

and then export the RIS file for further use: 

![afbeelding](https://github.com/asreview/asreview/assets/49488769/18b86739-67a3-44b8-a1bb-b6d71bb37a89)

Or if you use EndNote, you can surely make 'Smart Groups' using the ASReview decisions and comments in the Note field.

**Comment by dchamn** [comment]
Thanks very much!

________________________________
